src/main/java/com/teragrep/cfe_39/consumers/kafka/DatabaseOutput.java

```diff
@@ -0,0 +1,394 @@
+/*
+   HDFS Data Ingestion for PTH_06 use CFE-39
+   Copyright (C) 2022  Fail-Safe IT Solutions Oy
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+
+package com.teragrep.cfe_39.consumers.kafka;
+
+import com.teragrep.cfe_39.Config;
+import com.teragrep.cfe_39.avro.SyslogRecord;
+import com.teragrep.cfe_39.consumers.kafka.queue.WritableQueue;
+import com.teragrep.cfe_39.metrics.topic.TopicCounter;
+import com.teragrep.cfe_39.metrics.DurationStatistics;
+import com.teragrep.rlo_06.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.function.Consumer;
+
+import java.nio.ByteBuffer;
+
+//  The kafka stream should first be deserialized using rlo_06 and then serialized again using avro and stored in HDFS.
+//  The target where the record is stored in HDFS is based on the topic, partition and offset. ie. topic_name/0.123456 where offset is 123456
+
+// The mock consumer is activated for testing using the configuration file: readerKafkaProperties.getProperty("useMockKafkaConsumer", "false")
+
+public class DatabaseOutput implements Consumer<List<RecordOffset>> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseOutput.class);
+    private final RFC5424Frame rfc5424Frame = new RFC5424Frame(false);
+
+    private final String table;
+
+
+    private final DurationStatistics durationStatistics;
+    private final TopicCounter topicCounter;
+
+    private long lastTimeCalled = Instant.now().toEpochMilli();
+
+    public static final String ANSI_RESET = "\u001B[0m";
```

Comment:
ansi colors are not acceptable in LOGGER output, because machine processing becomes harder

Reply:
Solved in commit a1fda67179df7e304fab6dc81ca8a270e68d3b8b